build/proj/CoreBuild.proj

```diff
@@ -29,15 +29,31 @@
     <CommonBuildProperty Include="CIBuild=$(CIBuild)" />
     <CommonBuildProperty Include="RepoRoot=$(RepoRoot)" />
     <CommonBuildProperty Include="VersionsPropsPath=$(RepoRoot)build\import\Versions.props" />
+    <CommonBuildProperty Include="QuickSrcBuild=$(QuickSrcBuild)" />
   </ItemGroup>
 
   <ItemGroup>
     <SolutionRestoreAndBuildProperty Include="@(CommonBuildProperty)"/>
     <SolutionRestoreAndBuildProperty Include="__DeployProjectOutput=$(Deploy)" Condition="'$(Deploy)' != ''" />
   </ItemGroup>
 
-  <Target Name="LogInitialMessage">
-    <Message Text="Building $(SolutionFileName) [$(Configuration)]...%0a" Importance="high" />
+  <Target Name="GenerateSolutionForQuickSrcBuild" BeforeTargets="Restore" Condition="'$(QuickSrcBuild)' == 'true'">
+    <PropertyGroup>
+      <SolutionName>Src</SolutionName>
+      <SolutionFolder>$(RepoRoot)src/</SolutionFolder>
+      <SolutionFileName>$(SolutionName).sln</SolutionFileName>
+      <SolutionPath>$(SolutionFolder)$(SolutionFileName)</SolutionPath>
+    </PropertyGroup>
+    <Message Text="Generating Solution file [$(SolutionFileName)]..." Importance="high" />
+
+    <Exec Command="dotnet --version" />
```

Comment:
Is this needed, or a leftover from the development process?

Reply:
It isn't needed, no. In previous projects, I've had this for simply logging purposes when I first use the dotnet CLI. It was more applicable for hosted DevOps images as they update and change versions of things. I can remove it here.

Reply:
Removed.